compiler/src/iree/compiler/Codegen/Common/MaterializeTuningSpecsPass.cpp

```diff
@@ -138,8 +139,22 @@ getDefaultTuningSpec(ModuleOp module,
 
   // Load the library through the codegen dialect so that we cache the parsed
   // module.
-  return dialect.getOrParseTransformLibraryModule(defaultTuningSpecName,
-                                                  *defaultTuningSpecSource);
+  FailureOr<ModuleOp> defaultTransformLibrary =
+      dialect.getOrParseTransformLibraryModule(defaultTuningSpecName,
+                                               *defaultTuningSpecSource);
+
+  if (failed(defaultTransformLibrary)) {
+    return module->emitError()
+           << "Failed to load  default tuning spec" << defaultTuningSpecName;
+  }
+
+#ifndef NDEBUG
+  if (failed(mlir::verify(*defaultTransformLibrary)))
```

Comment:
Instead of the check above, we can do this:
```suggestion
  if (succeded(defaultTransformLibrary) && failed(mlir::verify(*defaultTransformLibrary)))
```